.gitignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+.vscode/launch.json
```

Comment:
Lets try:
 - remove ignore for this file + remove this file + commit + push
 - add to ignore whole .vscode dir + commit + push

Reply:
fixed